store/sqlstore/channel_store.go

```diff
@@ -1558,6 +1558,21 @@ func (s SqlChannelStore) GetDeleted(teamId string, offset int, limit int, userId
 	return channels, nil
 }
 
+var channelMembersForTeamWithSchemeSelectQueryX = sq.Select(
```

Comment:
We need to use `s.getQueryBuilder()`. Then we don't need to do the 

```go
if s.DriverName() == model.DatabaseDriverPostgres {
		query = query.PlaceholderFormat(sq.Dollar)
	}
```

business any more :wink:

Reply:
The problem here is, that `getQueryBuilder` is part of the `SqlChannelStore`. Would it make sense to move `channelMembersForTeamWithSchemeSelectQueryX` into `SqlChannelStore` and initialize it in `newSqlChannelStore`?

Reply:
Yes!